### ISSUE TITLE
Cannot Install

### ISSUE THREAD

**Issue by wkaskie** [vitaliy-bobrov/ionic-img-cache#17]
Hi,

I'm getting an error trying to install:

```
npm install --save ionic-img-cache
npm ERR! code ETARGET
npm ERR! notarget No matching version found for imgcache.js@^1.2.0
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget 
npm ERR! notarget It was specified as a dependency of 'ionic-img-cache'
npm ERR! notarget 

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/waynekaskie/.npm/_logs/2017-07-05T01_33_04_881Z-debug.log
```

**Comment by vitaliy-bobrov** [comment]
@wkaskie 
Fixed in `ionic-img-cache` v1.2.2. Problem is that `imgcahce.js` v1.2.0 not published to NPM yet.
See https://github.com/chrisben/imgcache.js/issues/217 .

**Comment by wkaskie** [comment]
Thanks! I just reinstalled and it completed without issue!
